### PR TITLE
(role/dnscache) update dns forwarder ip

### PR DESCRIPTION
**Pull request by dtapiacl**
This PR is to update the DNS forwarder from Crux to DC-LS-2.

### CODE REVIEW

File: hieradata/role/dnscache.yaml [hunk 1]
```diff
@@ -42,10 +42,10 @@ dns::zones:
     zonetype: "forward"
     forward: "only"
     forwarders:
-      - &crux "139.229.136.23"
-  "136.229.139.in-addr.arpa":
+      - &dc-ls-2 "139.229.135.58"
+  "135.229.139.in-addr.arpa":
```
Comment (jhoblitt) [inline hieradata/role/dnscache.yaml]: I don't think 139.229.135/24 is correct.

**Review by jhoblitt — CHANGES_REQUESTED** [review]

File: hieradata/role/dnscache.yaml (hunk 1 above)
Comment (jhoblitt) [reply hieradata/role/dnscache.yaml]: I still think the reverse zone is incorrect.

**Review by jhoblitt — CHANGES_REQUESTED** [review]
Please fix the reverse zone.

File: hieradata/role/dnscache.yaml [hunk 2]
```diff
@@ -42,10 +42,10 @@ dns::zones:
     zonetype: "forward"
     forward: "only"
     forwarders:
-      - &crux "139.229.136.23"
-  "136.229.139.in-addr.arpa":
+      - &dc-ls-2 "139.229.135.58"
+  "58.135.229.139.in-addr.arpa":
```
Comment (jhoblitt) [inline hieradata/role/dnscache.yaml]: This is still incorrect.  139.229.135/24 has IT chile servers in it.  We can not delegate this subnet to the PMO.

**Review by jhoblitt — CHANGES_REQUESTED** [review]
Please delegate the correct subnet.

File: hieradata/role/dnscache.yaml (hunk 2 above)
Comment (csilva-cl) [reply hieradata/role/dnscache.yaml]: I created the pmo records in r53. The delegation can be removed.

**Review by jhoblitt — APPROVED** [review]
Looks reasonable.

**Pull request merged**